File: typespec/problem/admin/authorization.go
package admin

import "github.com/vetchium/src/typespec/problem"

var AdminPermissionRequiredError = problem.Details{
	Type:   "vetchium-problem-details/admin-permission-required",
	Title:  "Admin permission required",
	Status: 403,
	Detail: "The authenticated admin lacks the required permission",
}

File: typespec/problem/admin/authorization.ts
import type { Details } from "../details.ts";

export const AdminPermissionRequiredError: Readonly<Details> = {
  type: "vetchium-problem-details/admin-permission-required",
  title: "Admin permission required",
  status: 403,
  detail: "The authenticated admin lacks the required permission",
};
